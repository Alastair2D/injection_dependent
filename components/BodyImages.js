import React, { Component } from "react";
import { Image } from "react-native";

export default class BodyImages extends Component {


  render() {
    return(
      <div><Text>Hello World</Text></div>
    )
  }
}
