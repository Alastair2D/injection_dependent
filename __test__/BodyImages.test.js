import { shallow } from 'enzyme';
import React from 'react';
import { Image } from 'react-native';
import BodyImages from '../components/BodyImages';

describe('BodyImages', () => {
  it('renders an image', () => {
    const bi = shallow(<BodyImages />);

    expect(bi.containsMatchingElement(<Image />)).toEqual(true);
  });

  it('renders correctly', () => {
    const bI = shallow(<BodyImages />);

    expect(bI).toMatchSnapshot();
  });
});
