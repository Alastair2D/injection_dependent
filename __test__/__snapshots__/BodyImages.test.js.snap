// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BodyImages renders correctly 1`] = `
<Component>
  <Image
    style={
      Object {
        "alignSelf": "center",
        "height": 280,
        "padding": 10,
        "width": 180,
      }
    }
  />
</Component>
`;
